ports/krb5/portfile.cmake

```
@@ -76,11 +76,16 @@ if (VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
         endforeach()    
     endif()
 else()
+    if(VCPKG_TARGET_IS_OSX)
+        vcpkg_replace_string("${SOURCE_PATH}/src/build-tools/mit-krb5.pc.in" "-lkrb5" "-framework Kerberos -lkrb5")
```

For traditional linkers, the independent library comes last. (XCode should use a moder linker, but some users might use different toolchains.)
```suggestion
        vcpkg_replace_string("${SOURCE_PATH}/src/build-tools/mit-krb5.pc.in" "-lkrb5" "-lkrb5 -framework Kerberos")
```

---

done.